docs/developers/keystore.md

```diff
@@ -0,0 +1,56 @@
+---
+id: keystore
+title: MultiversX Wallet - Keystore files
+---
+
+[comment]: # (mx-abstract)
+
+The keystore is a JSON file that encrypts a mnemonic (seed phrase). Until the user uses their password, the seed phrase remains encrypted, ensuring the safety and security of their transactions. The MultiversX keystore file provides users with a reliable and convenient method for managing their keys, protecting their assets, and ensuring the integrity of their transactions.
+
+[comment]: # (mx-context-auto)
+
+## MultiversX Keystore
+
+**How does a MultiversX Keystore look like?**
+
+Here is an example:
+
+```json
+{
```

Comment:
Please use the new format, instead of the old one.

https://raw.githubusercontent.com/multiversx/mx-sdk-js-wallet/main/src/testdata/withDummyMnemonic.json

Reply:
I am thinking about adding both